openbas-api/src/main/java/io/openbas/service/ReportService.java

```diff
@@ -50,6 +53,24 @@ public Report updateReport(@NotNull final Report report, @NotNull final ReportIn
         return this.reportRepository.save(report);
     }
 
+    public List<ReportInjectComment> updateReportInjectComment(@NotNull final Report report, @NotNull final Inject inject, @NotNull final ReportInjectCommentInput input){
+        ReportInjectComment reportInjectComment = report.getReportInjectsComments().stream()
+                .filter(c -> c.getInject().getId().equals(input.getInjectId()))
+                .findFirst()
+                .orElse(null);
```

Comment:
Can we have the direct query to do that ? 
To avoid hibernate to lazy load too much datas